src/engine/renderer/tr_shader.cpp

```diff
@@ -1733,15 +1734,17 @@ struct extraMapParser_t
 {
 	const char *suffix;
 	const char *description;
-	void ( *parser ) ( shaderStage_t*, const char** );
+	void ( *parser ) ( shaderStage_t*, const char**, int );
+	int bundleIndex;
 };
 
 static const extraMapParser_t extraMapParsers[] =
 {
-	{ "norm",    "normal map",     ParseNormalMap     },
-	{ "gloss",   "specular map",   ParseSpecularMap   },
-	{ "glow",    "glow map",       ParseGlowMap       },
-	{ "luma",    "glow map",       ParseGlowMap       },
+	// currently uses the collapse code path
+	{ "norm",    "normal map",     ParseNormalMap,     TB_COLORMAP, },
```

Comment:
In what circumstance would you want to use a different value besides TB_COLORMAP?

Reply:
When you do this:

```
textures/collection/name
{
	qer_editorImage textures/collection/name_d
	{
		diffuseMap textures/collection/name_d
		glowMap textures/collection/name_g
	}
}
```

The engine create a new stage with `ST_COLLAPSED_lighting_PBR` type, and adds `rock_d` texture to `TB_DIFFUSEMAP` and `rock_g` texture to `TB_GLOWMAP`.
And stage is ready to feed the renderer.

When you do this:

```
textures/collection/name
{
	qer_editorImage textures/collection/name_d

	diffuseMap textures/collection/name_d
	glowMap textures/collection/name_g
}
```

or this:

```
textures/collection/name
{
	qer_editorImage textures/collection/name_d
	{
		map textures/collection/name_d
		stage diffuseMap
	}
	{
		map textures/collection/name_g
		stage glowMap
	}
}
```

There is two stages created: one with `ST_DIFFUSEMAP` type with `rock_d` in `TB_COLORMAP`, and one with `ST_NORMLALMAP` type with `rock_g` in `TB_COLORMAP`.

Then a collapsing heuristing creates a new stage with  `ST_COLLAPSED_lighting_PBR` type and stores `rock_d` texture to `TB_DIFFUSEMAP` and `rock_g` texture to `TB_GLOWMAP`, initial stages are disabled.

We can't store `rock_d` and `rock_g` in `TB_DIFFUSEMAP` or `TB_GLOWMAP` in standalone stages in case we can't collapse.

If for some if we can't collapse, the glow map will be loaded as a color map with a blend function, like if it was initially written this way:

```
textures/collection/name
{
	qer_editorImage textures/collection/name_d

	diffuseMap textures/collection/name_d
	{
		map textures/collection/name_g
		blendfunc add
	}
}
```

We have to keep in mind we are attempting to collapse colormaps that has known behaviour to optimize the renderer, but they can all be rendered as separate colormap stages with proper blend function. If we fail to collapse, there must be good old standard colormap stages with their respective blend functions.